x-pack/qa/password-protected-keystore/src/javaRestTest/java/org/elasticsearch/password_protected_keystore/ReloadSecureSettingsWithPasswordProtectedKeystoreRestIT.java

```diff
@@ -37,12 +37,12 @@ public class ReloadSecureSettingsWithPasswordProtectedKeystoreRestIT extends ESR
         .nodes(NUM_NODES)
         .keystorePassword(KEYSTORE_PASSWORD)
         .name("javaRestTest")
+        .keystore(nodeSpec -> Map.of("xpack.security.transport.ssl.secure_key_passphrase", "transport-password"))
```

Comment:
This should work without the lambda I think, i.e. just `.keystore("xpack.security.transport.ssl.secure_key_passphrase", "transport-password")`